puppetdb/templates/prometheus-puppetdb-exporter-servicemonitor.yaml

```diff
@@ -0,0 +1,23 @@
+---
+{{ if .Values.prometheusPuppetdbExporter.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "prometheus-puppetdb-exporter.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "prometheus-puppetdb-exporter.name" . }}
+    helm.sh/chart: {{ include "puppetdb.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  endpoints:
+  - path: /metrics
+    port: puppetdb-exp
+  namespaceSelector:
+    matchNames:
+    - is-puppet
```

Comment:
Should be `{{ .Release.Namespace | quote }}` (probably my mistake though...)

Reply:
yes and better because the puppet namespace is not always named is-puppet